src/argilla/client/client.py

```diff
@@ -171,6 +172,30 @@ def __init__(
 
         self._check_argilla_versions()
 
+    def __repr__(self):
+        user = self._user
+        first_name = user.first_name if user.first_name != "" else None
+        ws_ds_dictionary = {}
+        for work_space in user.workspaces:
+            ws_ds_dictionary[work_space] = [
+                ds.name
+                for ds in datasets_api.list_datasets(client=self.http_client).parsed
+                if ds.workspace == work_space
+            ]
+        indent = "   "
+        return (
+            "ArgillaClient("
+            + textwrap.indent(f"\nusername='{user.username}'", indent)
+            + textwrap.indent(f"\nrole='{user.role}'", indent)
+            + textwrap.indent(f"\nfirst_name='{first_name}'", indent)
+            + textwrap.indent(f"\nlast_name='{user.last_name}'", indent)
+            + textwrap.indent(f"\npassword='***'", indent)
+            + textwrap.indent(f"\napi_key='***'", indent)
+            + textwrap.indent(f"\nworkspaces={user.workspaces}", indent)
+            + textwrap.indent(f"\ndatasets={ws_ds_dictionary}", indent)
+            + textwrap.indent(f"\nupdated_at='{user.updated_at}'", indent)
+        )
```

Comment:
I am missing a closing bracket in this tring

Reply:
also @kursathalat in the printed example above, there seem to be 8 spaces, but I prefer to only show 4 if that is the case.

Reply:
solved, but currently dataset_api.list_datasets not working probably bc of version, let me have a look at it.